docs/user/trusted-publishers/creating-a-project-through-oidc.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Creating a PyPI project with a trusted publisher
+---
+
+{{ preview('oidc-publishing') }}
+
+# Creating a PyPI project with a trusted publisher
+
+Trusted publishers are not just for pre-existing PyPI projects: you can also use
+them to *create* a PyPI project!
+
+This again reduces the number of steps needed to set up a fully automated PyPI
+publishing workflow: rather than having to manually upload a first release
+to "prime" the project on PyPI, you can configure a "pending" publisher
+that will *create* the project when used for the first time. "Pending"
+publishers are converted into "normal" publishers on first use, meaning that
+no further configuration is required.
+
+The process for configuring a "pending" publisher are similar to those for
+a normal publisher, except that the page is under your account sidebar
+instead of any project's sidebar (since the project doesn't exist yet):
+
+![](/assets/publishing-link.png)
+
+Clicking on "publishing" will bring you to the following form:
+
+![](/assets/pending-publisher-form.png)
+
+This form behaves the same as with publishers for existing projects, except that you
+also need to provide the name of the PyPI project that will be created.
+
+For example, if you have a repository at `https://github.com/example/awesome`
```

Comment:
Replace this with sampleproject too?

Just two weeks ago we had a report that someone had registered a made up domain name used in the CPython docs to host something sketchy! 
https://github.com/python/cpython/issues/102627

Reply:
Agreed!

Reply:
Sounds good -- I'll use `octo-org/octo-repo` as the slug here, since it's what GitHub's own docs use (and `octo-org` is controlled by them.)

Reply:
Done.